### PR TITLE
Update chaos-mesh extension to v1.2.3. Add AwsChaos

### PR DESCRIPTION
**Pull request by hypnoce**
## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [ ] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] Code contributed by me aligns with current project license: [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)
 - [x] I Added [CHANGELOG](../CHANGELOG.md) entry regarding this change
 - [ ] I have implemented unit tests to cover my changes
 - [ ] I have added/updated the [javadocs](https://www.javadoc.io/doc/io.fabric8/kubernetes-client/latest/index.html) and other [documentation](https://github.com/fabric8io/kubernetes-client/blob/master/doc/CHEATSHEET.md) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=fabric8io_kubernetes-client) report
 - [x] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

**Comment by hypnoce** [comment]
Java 8 build passed locally on my side, not sure why the CI is failing.

**Review by rohanKanojia — APPROVED** [review]

File: extensions/chaosmesh/generator/go.mod [hunk 1]
```diff
@@ -1,9 +1,9 @@
 module github.com/fabric8io/kubernetes-client/extensions/chaosmesh/generator
 
 require (
-	github.com/chaos-mesh/chaos-mesh v1.1.1
+	github.com/chaos-mesh/chaos-mesh/api/v1alpha1 v0.0.0-20210728073817-92f5e71c799f
```
Comment (manusa) [inline extensions/chaosmesh/generator/go.mod:4]: Is it possible to add a more stable reference (e.g. v1.2.3)?

https://github.com/chaos-mesh/chaos-mesh/commit/92f5e71c799f

Comment (hypnoce) [reply extensions/chaosmesh/generator/go.mod:4]: Let me check this

Comment (manusa) [reply extensions/chaosmesh/generator/go.mod:4]: Let's merge for now, eventually this will have to be updated to v2.0.0+

**Review by manusa — APPROVED** [review]
LGTM, thx!

**Pull request merged**